app/shared/reservation-confirmation/_confirm-reservation-modal.scss

```diff
@@ -1,7 +1,7 @@
 .app-ConfirmReservationModal {
   .modal-dialog {
     @media (min-width: $screen-sm-min) {
-      width: 700px;
+      width: 800px;
```

Comment:
$screen-sm-min is 768px, so you should not create bigger widths than that here. I think creating a new media query for $screen-md-min and setting the new width there would be the correct thing to do.